tests/testChecker.ml

```diff
@@ -197,6 +197,250 @@ let suite =
+       let _, checker = Checker.entrypoint_create_burrow (checker, (Ligo.nat_from_literal "0n", None)) in
+       Ligo.Tezos.new_transaction ~seconds_passed:10 ~blocks_passed:1 ~sender:alice_addr ~amount:(Ligo.tez_from_literal "0mutez");
+       let _, checker = Checker.entrypoint_mint_kit (checker, (Ligo.nat_from_literal "0n", (kit_of_mukit (Ligo.nat_from_literal "10_000_000n")))) in
+       (* Add some liquidity to the contract *)
+       Ligo.Tezos.new_transaction ~seconds_passed:1 ~blocks_passed:1 ~sender:alice_addr ~amount:(Ligo.tez_from_literal "0mutez");
+       let _, checker = Checker.entrypoint_add_liquidity
+           (checker,
+            (* Note: all values here were arbitrarily chosen based on the amount of kit we mint above *)
+            ( ctez_of_muctez (Ligo.nat_from_literal "5_000_000n")
+            , kit_of_mukit (Ligo.nat_from_literal "5_000_000n")
+            , lqt_of_denomination (Ligo.nat_from_literal "5_000_000n")
+            , Ligo.timestamp_from_seconds_literal 999
+            )
+           ) in
+       (* Now remove the liquidity *)
+       Ligo.Tezos.new_transaction ~seconds_passed:1 ~blocks_passed:1 ~sender:alice_addr ~amount:(Ligo.tez_from_literal "0mutez");
+       let ops, _ = Checker.entrypoint_remove_liquidity
+           (checker,
+            (* Note: all values here were arbitrarily chosen based on the amount of kit we mint above *)
+            ( lqt_of_denomination (Ligo.nat_from_literal "5_000_000n")
+            , ctez_of_muctez (Ligo.nat_from_literal "5_000_000n")
+            , kit_of_mukit (Ligo.nat_from_literal "5_000_000n")
+            , Ligo.timestamp_from_seconds_literal 999
+            )
+           ) in
+
+       let expected_ops = [
+         (LigoOp.Tezos.fa12_transfer_transaction
+            Fa12Interface.(
+              {address_to=alice_addr;
+               address_from=checker_address;
+               value=(Ligo.nat_from_literal "5_000_000n")}
+            )
+            (Ligo.tez_from_literal "0mutez")
+            (Option.get (LigoOp.Tezos.get_entrypoint_opt "%transfer" checker.external_contracts.ctez))
+         );
+       ] in
+       assert_operation_list_equal ~expected:expected_ops ~real:ops
+    );
+
+    ("entrypoint_touch - emits expected operations when checker needs to be touched" >::
+     fun _ ->
+       Ligo.Tezos.reset ();
+       let checker = empty_checker in
+       Ligo.Tezos.new_transaction ~seconds_passed:1 ~blocks_passed:1 ~sender:alice_addr ~amount:(Ligo.tez_from_literal "0mutez");
+       let ops, _ = Checker.entrypoint_touch (checker, ()) in
+
+       let expected_ops = [
+         (LigoOp.Tezos.nat_contract_transaction
+            (Option.get (LigoOp.Tezos.get_entrypoint_opt "%receive_price" !Ligo.Tezos.self_address))
+            (Ligo.tez_from_literal "0mutez")
+            (Option.get (LigoOp.Tezos.get_entrypoint_opt "%getPrice" checker.external_contracts.oracle))
```

Comment:
Might be slightly more robust if you use the `get_oracle_entrypoint` here instead of mentioning `%getPrice` directly. Something along these lines:
```suggestion
            (Checker.get_oracle_entrypoint checker.external_contracts)
```

Reply:
Nice, thanks for the tip. I didn't realize that helper existed.